samples/README.md

```diff
@@ -0,0 +1,49 @@
+### Sample programs to demonstrate SGX-LKL functionality
+
+1. Basic
+   
+   a. [Attack](basic/attack) \
+        This sample demonstrates how memory contents can be easily read when running in regular Docker but are protected in hardware enclaves.
+        Test the two scenarios and see how i) SGX-LKL protects data as compared to regular containers and ii) SGX-LKL hardware provides protection to the data in it
+
+   b. [Helloworld](basic/helloworld) \
+        Cross-compile a C application for running with SGX-LKL
+        Simple hello world C program run in sgx-lkl
+    
+2. Containers
+   
+    a. [Alpine](containers/alpine) \
+        Miniroot is a quick way of building a small Alpine disk image, appropriate for use with sgx-musl hosting redis.
```

Comment:
Do you mean sgx-lkl or sgx-musl?

Reply:
Fixed the mixup in language, thanks for pointing that out!